MD/md-3/README.md

```diff
@@ -0,0 +1,81 @@
+# MD-3: MCR under Network Partitions and Asynchrony
+- **Description**: Provide a model for MCR and suggest a mechanism for handling network partitions and asynchrony.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+
+
+## Overview
+MCR provides a stake-based settlement mechanism used in Movement Lab's Suzuka Network. In it's current form, it does not take any steps to account for network partitions or, more generally, asynchrony.
+
+Henceforth, we will simply refer to network partitions, as all forms of asynchrony with which we are concerned can be modeled as such. Even if a node is not technically offline, the remainder of this disiderata will be concerned with the case in which it is not able to commit meaningfully to the current block height.
```

Comment:
Yeah, I would say it could for logical reasons--not just network delays--not be able to commit in agreement with the network. That's the heart of the upgrades problem here. That's why I went with "commit meaningfully."

Reply:
mhm.. still think "meaningfully" is not clear.